controller/lib/core/sensors.h

```diff
@@ -28,6 +28,16 @@ Arduino Nano and the MPXV5004GP and MPXV7002DP pressure sensors.
 #include "units.h"
 #include <optional>
 
+// Structure for strongly typed sensor readings, denoted as Values, as opposed
+// to Readings which are generically typed to be readable in serial link
```

Comment:
I appreciate this mnemonic, but it's not working for me, lol.

How about SensorReadings and SensorReadingsProto?  (Or SensorValues/SensorValuesProto.)  It's a little annoying because none of our protos are named FooProto, but since these are basically the same thing, I feel like anything other than FooProto is going to be ambiguous and hard to remember.